sinks/gcm/driver.go

```diff
@@ -384,3 +256,42 @@ func sendRequest(url string, token string, request interface{}) error {
 
 	return nil
 }
+
+func (self *gcmSink) DebugInfo() string {
+	return "Sink Type: GCM"
+}
+
+// Returns a thread-compatible implementation of GCM interactions.
+func NewSink() (sink_api.ExternalSink, error) {
+	// TODO: Document why this sleep is necessary.
+	time.Sleep(3 * time.Second)
```

Comment:
LOL this slipped in, I saw flaked with OnGCE() so we should probably try it for ~15s before quitting. TODO is fine